js/gladiator.js

```diff
@@ -0,0 +1,21 @@
+
+
+
+
+   export default class Gladiator{
+
+        constructor(name,weapon){
+     
+          this.name = name;
+          if(weapon = ("Spear" ||  "Club" ||"Trident")){
```

Comment:
invalid logical statement, it will always reassign weapons input to "Spear",
a proper way to write it would be `if(weapon == "Spear" ||  weapon == "Club" || weapon == "Trident"))`